src/queue/workers.ts

```diff
@@ -148,3 +149,43 @@ export const expirePing = async (pingId: string) => {
     console.log(error.message);
   }
 };
+
+/**
+ * Remove a group invite notification from the database after the queue job has been processed
+ * @param {string} userId - The ID of the user who received the notification
+ * @param {string} groupId - The ID of the group that the notification was for
+ */
+export const removeGroupInviteNotification = async (
+  userId: string,
+  groupId: string
+) => {
+  try {
+    await Notification.findOneAndDelete({
+      userId: userId,
+      notificationType: NotificationType.friendRequest,
```

Comment:
I believe it's this: 
```suggestion
      notificationType: NotificationType.groupInvite,
```